tests/drivers/fuel_gauge/sbs_gauge/testcase.yaml

```diff
@@ -3,6 +3,13 @@ tests:
   drivers.sbs_gauge_new_api:
     tags: test_framework
     filter: dt_compat_enabled("sbs,sbs-gauge-new-api")
-    platform_allow: native_posix qemu_cortex_a9 nucleo_f070rb qemu_arc_hs
+    extra_args:
+      CONF_FILE="prj.conf;boards/emulated_board.conf"
+      DTC_OVERLAY_FILE="boards/emulated_board.overlay"
+    platform_allow:
```

Comment:
Droop the platform allow now that you have a common emulated overlay, should work everywhere.

Reply:
```
INFO    - The following issues were found (showing the top 10 items):
INFO    - 1) tests/drivers/fuel_gauge/sbs_gauge/drivers.sbs_gauge_new_api on qemu_cortex_a53_smp error (Cmake build failure)
INFO    - 2) tests/drivers/fuel_gauge/sbs_gauge/drivers.sbs_gauge_new_api on qemu_cortex_a53 error (Cmake build failure)
INFO    - 3) tests/drivers/fuel_gauge/sbs_gauge/drivers.sbs_gauge_new_api on qemu_malta_be failed (Failed)
INFO    - 4) tests/drivers/fuel_gauge/sbs_gauge/drivers.sbs_gauge_new_api on qemu_leon3 failed (Failed)
INFO    - 
INFO    - To rerun the tests, call twister using the following commandline:
```

Running `twister -c -v -i -T tests/drivers/fuel_gauge/` seems to work on all but 3 (technically 4) qemu boards. That might take quite a bit of debugging which I'd prefer to do in some other PR with added TODO for these specifically. It looks like `qemu_cortex_a53*` boards seem to just need a special case overlay to build.

Do we have a way to disallow just these platforms? I could keep the `platform_allow` for all the ones that passed but that seems verbose.

Reply:
platform_exclude can exclude specific platforms, but personally I'd spend some time looking at the issues, they can be useful in catching problems with your API

Reply:
It did catch a bug in my emulator :stuck_out_tongue: